seeds/gmx/avalanche_c/gmx_glp_aum_seed.csv

```diff
@@ -0,0 +1,6 @@
+date,usdc_aum,usdc_e_aum
+2022-09-15,20359495.092885,18949462.097249
+2022-09-16,20327050.875974,18440021.732571
```

Comment:
Hi @theachenyj, please have a look on this line.
Moreover this is your [test query result](https://dune.com/queries/1707843?d=1)
<img width="1007" alt="image" src="https://user-images.githubusercontent.com/6568873/205596461-13b8ecca-6338-4088-bc77-51b1f19a7976.png">
It seems there is a null value for this record when joining.

Reply:
Finally! I've made a mistake in aum cte. Thx a lot.